src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+  
+    <form 
+      class="form"
+      method="get"
+      action="/"
+      data-qa="big"
+    >
+      <img
+        class="icon"
+        src="images/Search.svg" 
+        alt="Search Image"
+      > 
+      <label for="search-input">  
+        <input 
+            class="input input--big"
+            type="text"
+            name="search"
+            id="search-input"
+            placeholder="Try &quot;Los Angeles&quot;"
+            data-qa="keypress"
+        >
+      </label>
```

Comment:
Pay attention on indents
```suggestion
      <label for="search-input">  
        <input 
          class="input input--big"
          type="text"
          name="search"
          id="search-input"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
      </label>
```